doc/source/whatsnew/v2.0.0.rst

```diff
@@ -1423,6 +1423,15 @@ Metadata
 
 Other
 ^^^^^
+
+.. note::
```

Comment:
This seems to be a bit strange location to put it? (inside a "Bugs" subsection)

Reply:
Yeah good point, but did not want to put it right at the top, so not really sure where to place it

Reply:
We could put it under other api changes?